test-cases/tests/dashboards/e07-verify-drill-down-links.md

```diff
@@ -15,6 +15,7 @@ products:
     targets:
       - 0.1.0
       - 0.2.0
+      - 1.0.0
```

Comment:
Does this file not need to be split up into a and b ?

Reply:
yes I missed that, will fix it in a next commit, thanks